lib/webpacker/runner.rb

```diff
@@ -0,0 +1,34 @@
+require "shellwords"
+
+module Webpacker
+  class Runner
+    def self.run(argv)
+      $stdout.sync = true
+
+      new(argv).run
+    end
+
+    def initialize(argv)
+      @argv = argv
+
+      @app_path          = File.expand_path("../", __dir__)
```

Comment:
Does `__dir__` work correctly here (in a file in the gem)?

Reply:
No, good catch—error in local testing. I may need to pass the app_path into the runner which may mean keeping the install template with less code instead of using a bundler binstub.